exploration/bidi-usability.md

```diff
@@ -256,13 +258,15 @@ the user to set the base direction of a _literal_ or _pattern_ according to its
 actual contents.
 
 This would change the ABNF as follows:
-(Notice that this change includes a production `bidi` described further down
-in this document)
 ```abnf
-literal        = ( open-isolate (quoted / (unquoted [bidi])) close-isolate)
-               / (quoted / (unquoted [bidi]))
-quoted-pattern = ( open-isolate "{{" pattern "}}" close-isolate)
-               / ("{{" pattern "}}")
+name           = (open-isolate name-body close-isolate) / name-body
```

Comment:
This is a problem because `name` is used to build a variety of other constructs (`variable`, `reserved-keyword`, `identifier`, etc.). This change puts the isolates _inside_ these constructs, e.g. `$\u2066name\u2069` rather than on the outside.

This will make it harder for implementations, since they can't take the parsed token and compare it immediately. They have to stop to remove isolates. My original design avoided this problem by making the isolates not parse into names/identifiers/tokens.